sketch/RFUT_data.hpp

```diff
@@ -27,7 +27,7 @@ struct RFUT_data_t {
     RFUT_data_t (int N, skylark::base::context_t& context)
         : _N(N), _creation_context(context) {
 
-        build();
+//        build();
```

Comment:
Yes, `_` followed by an upper case letter are reserved identifiers (according to the C99 standard). I personally don't like the aesthetics of `name_.something()` but I agree with you that clashing with the standard is bad. We can and should tweak our coding style, but not in this pull request.

*Note*: ending names with `_t` is another issue (reserved in the POSIX standard).